mmdeploy/codebase/mmdet/core/bbox/distance_point_bbox_coder.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import mmdet.core.bbox.transforms
+
+from mmdeploy.core import FUNCTION_REWRITER
+
+
+@FUNCTION_REWRITER.register_rewriter(
+    func_name='mmdet.core.bbox.coder.DistancePointBBoxCoder.decode',
+    backend='default')
+def decode__default(ctx, self, points, pred_bboxes, max_shape=None):
```

Comment:
This naming style is not aligned with others. Besides, should add ut for this rewriting.